src/coreclr/jit/llvm.cpp

```diff
@@ -1593,6 +1677,9 @@ void Llvm::visitNode(GenTree* node)
         case GT_STOREIND:
             importStoreInd((GenTreeStoreInd*)node);
             break;
+        case GT_STORE_OBJ:
+            importStoreObj((GenTreeStoreInd*)node);
+            break;
```

Comment:
```suggestion
        case GT_STOREIND:
            importStoreInd(node->AsStoreInd());
            break;
        case GT_STORE_OBJ:
            importStoreObj(node->AsStoreInd());
            break;
```
(Would also be nice to rename these `buildStoreXYZ` for consistency, or at least the new function)